Documentation/ExtensionArchitecture/ComposerJson/Index.rst

```diff
@@ -274,24 +270,9 @@ Example for extension key **bootstrap_package**:
       }
    }
 
-
-As all extensions available in the TER can be installed
-with `composer require typo3-ter/ext-key`, this makes sure that
-there will be no conflicts with packages installed or required
-via Packagist or from another source.
-
-Since the TER Composer repository is deprecated and not all extensions
-must be available in TER, this property is usually not required.
-
-Properties no longer used
-=========================
-
-version
--------
-
-Was used in earlier TYPO3 versions.
-For TYPO3 versions 7.6 and above you should not use the version property.
-The version for the extension is set in the file :ref:`ext_emconf.php <ext_emconf-php>`.
+This was used previously as long as the TER Composer Repository was 
+relevant. Since the TER Composer Repository is deprecated, this property 
+is not required.
```

Comment:
```suggestion
relevant. Since the TER Composer Repository is deprecated, the `typo3-ter/*` entry
within `replace` is not required.
```

We should keep in mind that there may be other intentional `replace` entries aside from TER.